locales/nl-NL.yml

```
@@ -470,52 +551,404 @@ promote: "Promoot"
 hardWordMute: "Harde woorddemping"
 showMutedWord: "Gedempte woorden weergeven"
 hardWordMuteDescription: "Verbert notities die het gespecificeerde woord of zin bevatten. In tegenstelling tot woorddemping wordt de notitie volledig verborgen."
+regexpError: "Fout in reguliere expressie"
+regexpErrorDescription: "Er is een fout opgetreden in de reguliere expressie op regel {line} van uw {tab} woord dempen:"
+instanceMute: "Instantie dempers"
+userSaysSomething: "{name} zei iets"
 userSaysSomethingAbout: "{name} zei iets over '{word}'"
+makeActive: "Activeren"
+display: "Weergave"
+copy: "Kopiëren"
 copiedToClipboard: "Naar het klembord gekopieerd"
+metrics: "Metrieken"
+overview: "Overzicht"
+logs: "Protocollen"
+delayed: "Vertraagd"
+database: "Database"
+channel: "Kanalen"
+create: "Creëer"
+notificationSetting: "Instellingen meldingen"
+notificationSettingDesc: "Selecteer het type meldingen dat moet worden weergegeven."
+useGlobalSetting: "Globale instelling gebruiken"
+useGlobalSettingDesc: "Als deze optie is ingeschakeld, worden de meldingsinstellingen van je account gebruikt. Als deze optie uitgeschakeld is, kunnen individuele configuraties worden gemaakt."
+other: "Ander"
+regenerateLoginToken: "Login token opnieuw genereren"
+regenerateLoginTokenDescription: "Regenereren van het token dat intern wordt gebruikt om in te loggen. Dit is normaal gezien niet nodig. Alle apparaten worden afgemeld tijdens het regenereren."
 theKeywordWhenSearchingForCustomEmoji: "Dit is het keyword dat gebruikt wordt bij het zoeken naar eigen emojis."
+setMultipleBySeparatingWithSpace: "Scheid elementen met een spatie om meerdere instellingen te configureren."
+fileIdOrUrl: "Bestands-ID of URL"
+behavior: "Gedrag"
+sample: "Voorbeeld"
+abuseReports: "Meldt"
+reportAbuse: "Meld"
+reportAbuseRenote: "Meld renote"
+reportAbuseOf: "Meld {name}"
 fillAbuseReportDescription: "Vul s.v.p. de details in over deze melding. Geef, als het over een specifieke notitie gaat, ook de URL op."
+abuseReported: "Uw rapport is verzonden. Hartelijk dank."
+reporter: "Verslaggever"
+reporteeOrigin: "Oorsprong van de gemelde persoon"
+reporterOrigin: "Verslaggever Oorsprong"
+send: "Stuur"
+openInNewTab: "In nieuw tabblad openen"
+openInSideView: "In zijaanzicht openen"
+defaultNavigationBehaviour: "Standaard navigatie gedrag"
+editTheseSettingsMayBreakAccount: "Het wijzigen van deze instellingen kan je account beschadigen."
+instanceTicker: "Instantie-informatie van notities"
+waitingFor: "Wachten op {x}"
+random: "Willekeurig"
+system: "Systeem"
+switchUi: "UI omschakelen"
+desktop: "Desktop"
+clip: "Clip aanmaken"
+createNew: "Nieuwe aanmaken"
+optional: "Optioneel"
+createNewClip: "Nieuwe clip aanmaken"
+unclip: "Van clip verwijderen"
+confirmToUnclipAlreadyClippedNote: "Deze notitie is al toegevoegd aan de clip “{naam}”. Wil je deze uit deze clip verwijderen?"
```

```suggestion
confirmToUnclipAlreadyClippedNote: "Deze notitie is al toegevoegd aan de clip “{naam}”. Wil je deze uit deze clip verwijderen?"
```

これの {name}であるべきところが {naam}になってる

---

nlってなんだ

---

Nederlandか